libglusterfs/src/common-utils.c

```diff
@@ -75,6 +76,15 @@ char *vol_type_str[] = {
 typedef int32_t (*rw_op_t)(int32_t fd, char *buf, int32_t size);
 typedef int32_t (*rwv_op_t)(int32_t fd, const struct iovec *buf, int32_t size);
 
+char *xattrs_to_heal[] = {"user.",
```

Comment:
```suggestion
static char *xattrs_to_heal[] = {"user.",
```

Reply:
The change does not exist in devel branch.First i have to merge the changes in devel branch so  i will send a separate patch to handle this.